pulp_python/app/tasks/sync.py

```diff
@@ -50,8 +50,7 @@ def sync(remote_pk, repository_pk):
             remote_metadata = _fetch_remote(remote)
             remote_keys = set([content['filename'] for content in remote_metadata])
 
-            mirror = remote.sync_mode == 'mirror'
-            delta = _find_delta(inventory=inventory, remote=remote_keys, mirror=mirror)
+            delta = _find_delta(inventory=inventory, remote=remote_keys, mirror=True)
```

Comment:
i think remove `mirror` and its uses altogether, but this is fine if you prefer.

Reply:
I'll do that